tests/integration_tests/canal_json_storage_basic/run.sh

```diff
@@ -8,39 +8,13 @@ WORK_DIR=$OUT_DIR/$TEST_NAME
 CDC_BINARY=cdc.test
 SINK_TYPE=$1
 
-# start the s3 server
```

Comment:
Can I ask why do we need to remove it?

Reply:
we use [loacl storage in integration test to facilitate debug](https://github.com/pingcap/tiflow/pull/8751/files#diff-0dac0ea75b05049ecd2a171ef67193c96891fc2677187257e06314dea92343d7R30).

Reply:
Got it. Thanks!